src/java/com/phenix/pct/AnalyticsBuildListener.java

```diff
@@ -0,0 +1,100 @@
+package com.phenix.pct;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.Map.Entry;
+
+import org.apache.tools.ant.BuildEvent;
+import org.apache.tools.ant.BuildListener;
+import org.apache.tools.ant.Task;
+
+public class AnalyticsBuildListener implements BuildListener {
+    private static final Map<String, Integer> taskCount = new HashMap<>();
+
+    private boolean isPCTTask(Task task) {
+        if (task.getTaskType().toLowerCase().startsWith("pct"))
+            return true;
+        if ("DlcVersion".equalsIgnoreCase(task.getTaskType())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1126)